quote from http://www.cbcb.umd.edu/software/jellyfish/:
"JELLYFISH runs on 64-bit Intel-compatible processors running Linux or FreeBSD (including Intel Macs). It requires GNU GCC to compile."
--- trinityrnaseq_r2012-10-05/Makefile.orig	2012-12-14 12:09:09.285823000 +0100
+++ trinityrnaseq_r2012-10-05/Makefile	2012-12-14 12:09:31.595629000 +0100
@@ -20,7 +20,7 @@
                 && ./configure --prefix=`pwd` $(INCHWORM_CONFIGURE_FLAGS) && $(MAKE) install
 	cd Chrysalis && $(MAKE) UNSUPPORTED=yes $(CHRYSALIS_MAKE_FLAGS)
 	cd trinity-plugins/rsem && $(MAKE) 
-	cd trinity-plugins/jellyfish && ./configure --prefix=`pwd` && $(MAKE)
+	#cd trinity-plugins/jellyfish && ./configure --prefix=`pwd` && $(MAKE)
 	cd trinity-plugins/fastool && $(MAKE)
 	cd trinity-plugins/slclust && $(MAKE)
 	cd trinity-plugins/coreutils && ./build_parallel_sort.sh
